ckanext/canada/validators.py

```diff
@@ -270,19 +270,19 @@ def canada_org_title_translated_output(key, data, errors, context):
         data[new_key] = data[key]['en'] + ' | ' + data[key]['fr']
 
 
-def protect_is_federal(key, data, errors, context):
+def protect_reporting_requirements(key, data, errors, context):
     """
-    Ensure the is_federal flag is not changed by an unauthorized user.
+    Ensure the reporting_requirements field is not changed by an unauthorized user.
     """
     if not is_sysadmin(context['user']):
-        errors[key].append("is_federal field can only be changed by sysadmin")
+        errors[key].append("reporting_requirements field can only be changed by sysadmin")
         raise StopOnError
```

Comment:
I think you need to replace the value passed with the existing value when a user isn't sysadmin instead of returning an error, otherwise regular users won't be able to edit their own info (full name etc.)

Reply:
or maybe only return an error if the value submitted doesn't match the existing one for non-sysadmins